src/Fixer/Phpdoc/PhpdocIndentFixer.php

```diff
@@ -53,14 +53,6 @@ class DocBlocks
      */
     public function getPriority()
     {
-        /*
```

Comment:
lets keep for now, especially since we've no test that shows this

Reply:
We have: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/tests/AutoReview/FixerFactoryTest.php#L259

And it claims this is 3rd, not 2nd as in removed comment.

Reply:
We've indeed a test to make sure the priority is what it is, fair enough, however we've no test itself to show why this must be this way. The priority fixture test is missing, there is no prove that the priority must be like this, we've only this reasoning in the comment to go with. If we would've a priority test with sample code to fix that shows the need for the priority I'm happy to remove the comment.

Reply:
like these; https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/tests/AutoReview/FixerFactoryTest.php#L299

Reply:
Let's [fix](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4769) this then and go with current changes.

Reply:
great :+1: